xml/System.Diagnostics.Tracing/EventSource.xml

```diff
@@ -3505,6 +3505,106 @@ class AnotherEventSource : EventSource {
   
 ```  
   
+  These are the expected sizes and data encodings for standard serializable types:
+
+```csharp 
+// bool arg
+int temp = arg ? 1 : 0;
+desc.DataPointer = (IntPtr)(&temp);
+desc.Size = 4;
+
+// byte arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 1;
+
+// sbyte arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 1;
+
+// sbyte arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 1;
+
+// char arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 2;
+
+// short arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 2;
+
+// ushort arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 2;
+
+// int arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 4;
+
+// uint arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 4;
+
+// long arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 8;
+
+// ulong arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 8;
+
+// float arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 4;
+
+// double arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 8;
+
+// decimal arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 16;
+
+// Guid arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = 16;
+
+// IntPtr arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = IntPtr.Size;
+
+// UIntPtr arg
+desc.DataPointer = (IntPtr)(&arg);
+desc.Size = UIntPtr.Size;
+
+// DateTime arg
+long fileTime = arg.ToFileTimeUtc();
+desc.DataPointer = (IntPtr)(&fileTime);
+desc.Size = 8;
+
+// string arg
+fixed(byte* ptr = arg)
+{
+    int length = arg.Length;
+    desc.DataPointer = ptr;
+    desc.Size = (arg.Length+1)*2;
```

Comment:
Perhaps a comment explaining the conversion between multi-byte char `Length` and number of bytes `Size` here?